docs/OV_Runtime_UG/hetero_execution.md

```diff
@@ -1,70 +1,74 @@
 
-- The fallback automatic policy causes "greedy" behavior and assigns all operations that can be executed on certain device according to the priorities you specify (for example, `ov::device::priorities("GPU,CPU")`).
-Automatic policy does not take into account device peculiarities such as the inability to infer some operations without other special operations placed before or after that layer. The plugin is responsible for solving such cases. If the device plugin does not support the subgraph topology constructed by the HETERO device, then you should set affinity manually.
-- Manual policy assumes explicit setting of affinities for all operations in the model using the runtime information ov::Node::get_rt_info.
+These two steps are not interconnected and affinities can be set in either of two ways (as described below): in the `manual` mode or using the `automatic fallback` policy.
 
 ### Defining and Configuring the Hetero Device
 
-Following the OpenVINO™ convention of labeling devices, the Hetero execution uses the name `"HETERO"`. Configuration options for the Hetero device:
+Following the OpenVINO™ naming convention, the Hetero execution plugin is assigned the label of `"HETERO".` It may be defined with no additional parameters, resulting in defaults being used, or configured further with the following setup options: 
 
-| Parameter name | C++ property | Parameter values | Default | Description |
-| -------------- | ---------------- | ---------------- | --- | --- |
-| "MULTI_DEVICE_PRIORITIES" | `ov::device::priorities` | comma-separated device names with no spaces | N/A | Prioritized list of devices |
++-------------------------------+--------------------------------------------+-----------------------------------------------------------+
+| Parameter Name & C++ property | Property values                            | Description                                               |
++===============================+============================================+===========================================================+
+| | "MULTI_DEVICE_PRIORITIES"   | | HETERO: <device names>                   | | Lists the devices available for selection.              |
+| | `ov::device::priorities`    | | comma-separated, no spaces               | | The device sequence will be taken as priority           |
+| |                             | |                                          | | from high to low.                                       |
++-------------------------------+--------------------------------------------+-----------------------------------------------------------+
 
```

Comment:
why have we moved the table at the beginning? Table is a dry reference information while affinities, auto / manual modes are more interesting for users to understand HETERO capabilities.

Reply:
Moreover, the table is not rendered well. Please, download compiled version of the docs.

Reply:
The table hasn't moved but I wanted to combine information on auto / manual - some of it was before the table and the section with a header was below - I put the top in the below part :) I see no problem in switching places between the "policies for assigning affinities" and "configuring hetero."
And yes, obviously, I have removed the sphinx directive around the table. My mistake, corrected.